.cirrus.yml

```diff
@@ -15,6 +15,7 @@ lint_task:
 
 
 FreeBSD_task:
+  allow_failures: $PYTHON == 3.7
```

Comment:
Can I yell at FreeBSD about this? They don't have pip3.7 packages yet and 3.8 is around the corner?

Reply:
@astronouth7303 IIRC, it's just that there was some very recent shuffling around in how Python versions are handled in the ports tree (switching to using flavors of a single source port), which involves rebuilding all those packages, and most py37 packages haven't been rebuilt yet:
```
% pkg search ^py37 | wc -l
       6
% pkg search ^py36 | wc -l  
    2038
```

Reply:
Ahhh, so is there an ETA on those packages getting built?

Reply:
No idea offhand